src/app/(pages)/profile/cross-swap/_components/config.ts

```diff
@@ -0,0 +1,61 @@
+// Configuration
+export const CONFIG = {
+    API: {
+        BASE_URL: 'https://www.okx.com',
+        RETRY: { MAX_ATTEMPTS: 3, DELAY: 1000 },
+        HEADERS: {
+            'Content-Type': 'application/json',
+            'OK-ACCESS-KEY': process.env.NEXT_PUBLIC_OKX_API_KEY,
+            'OK-ACCESS-PASSPHRASE': process.env.NEXT_PUBLIC_OKX_PASSPHRASE,
+            'OK-ACCESS-PROJECT': process.env.NEXT_PUBLIC_OKX_PROJECT_ID,
+            'OK-ACCESS-SECRET': process.env.NEXT_PUBLIC_OKX_SECRET_KEY,
```

Comment:
If we add NEXT_PUBLIC prefix, those keys will be exposed. Since the client page requires NEXT_PUBLIC prefix for env variables, we need to add some server side logic where we use the keys, like we do with PRIVY private key and SUPABASE private key.